docs/site/Update-generator.md

```diff
@@ -0,0 +1,64 @@
+---
+lang: en
+title: 'Update generator'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Update-generator.html
+---
+
+{% include content/generator-create-app.html lang=page.lang %}
+
+### Synopsis
+
+The `lb4 update` command runs against a LoopBack 4 project and checks
+dependencies against the installed `@loopback/cli`. By default, it checks for
+exact match. Use `--semver` option to check compatibility using semver
+semantics.
+
+```sh
+lb4 update [options]
+```
+
+If the command is run without a LoopBack 4 project, it prints the version of the
+current CLI and versions of LoopBack packages that are released with the
+`@loopback/cli`. This is the same as `lb4 -v`.
+
+### Options
+
+`--semver` : _(Optional)_ Use semver semantics to check version compatibility
+for project dependencies of LoopBack modules
+
+### Output
+
+The following is a sample output from `lb4 update` against a LoopBack 4 project
+generated by older version of CLI:
+
+```
+The project was originally generated by @loopback/cli@<unknown>.
```

Comment:
I find the `<unknown>` version suspicious. Is it a temporary thing because the current CLI versions are not writing the version info? Can we print a nicer error message in such case? For example:

```suggestion
The project was originally generated by an older version of @loopback/cli.
```

Also what happens after we run `npm update` and then a new version of `@loopback/cli` is published - are we going to print the CLI version that was used to generate the project or the last CLI version that was executed to update dependencies?

Reply:
The `unknown` is only there because my very old project does not have such information.